addons/web/static/tests/legacy/control_panel/groupby_menu_tests.js

```diff
@@ -62,6 +67,7 @@ odoo.define('web.groupby_menu_tests', function (require) {
             assert.strictEqual(optionEls[1].innerText.trim(), 'Date');
             assert.strictEqual(optionEls[2].innerText.trim(), 'Foo');
             assert.strictEqual(optionEls[3].innerText.trim(), 'Many2Many');
+            assert.containsNone(controlPanel, '.o_add_custom_group_menu select option[value="m2m_not_stored"]');
```

Comment:
I would replace the strictEquals and containsNone with a deepEqual on all optionEls, so that it would be clear what are the options
assert.deepEqual([...optionEls].map((el) => el.innerText), ["Birthday, ...])

containsNone can too easily become useless